native/dll/DShowCapture-2/GraphManager.cpp

```diff
@@ -111,52 +112,59 @@ JNIEXPORT jlong JNICALL Java_sage_DShowCaptureDevice_initGraph0
 	strncpy( TuningMode, pTuningMode, sizeof(TuningMode) );
 	env->ReleaseStringUTFChars( jtuningMode, pTuningMode);
 
-	if ( (deviceCaps & BDA_CAPTURE_ALL) && BDATypeNum( deviceCaps ) > 0 ) ////ZQ REMOVE ME
+	DETECTED_CAPTURE_NUM = BDATypeNum(deviceCaps);		//Determine the number of BDA tuners 
```

Comment:
Change this to be:

int detectedCaptureNum = BDATypeNum(deviceCaps); // comment

And rename uses of DETECTED_CAPTURE_NUM below to be detectedCaptureNum.

Reply:
Done